### ISSUE TITLE
Host solidproject.org on a Solid server

### ISSUE THREAD

**Issue by csarven** [solid/solidproject.org#248]
Given the nature of this project, filing this as a bug as opposed to a feature request.

Eating own dogfood shouldn't be a debate..

**Comment by RubenVerborgh** [comment]
Sure, but whatever server we use in the back, we should still have good geo-distributed caching such that the website is fast in every location on earth. (With GitHub Pages, we get that automatically.)

Case in point: I'm all for decentralization and I independently host my webpage (some sections of which are Solid-based indeed), but I still add Cloudflare caching on top such that people from the US and Australia have the same experience.

TL;DR: let's not sacrifice universality for dogfooding.

**Comment by csarven** [comment]
Sure, but the issue did not specify the actual process or all the bits that are required or optional. Certainly there are many things to consider and we can list and prioritise.

As I see it, while it is important, not having "good geo-distributed caching" should be a show stopper to get this off the ground. If the suggestion is that it should be required to launch a self-hosted website using a Solid server, it'd be good to have analytics showing the delta between with and without such that we can conclude universality is sacrificed.

**Comment by csarven** [comment]
Having the website use what it describes consequently enables the community to engage and evolve.

For example, the "labs" document ( https://github.com/solid/solidproject.org/pull/244 ) with an inbox can accept notifications announcements or requests to be included in its index of research institutions working on Solid. A fundamental activity.

**Comment by RubenVerborgh** [comment]
> that'd be useful but not a showstopper

I can't parse what you mean with "that".

Do you mean "having it equally fast/accessible for everyone in the world" or "having it run on Solid"?

I can only support dogfooding when the experience is not substantially affected, and it would be if we don't put a Cloudflare or similar in the middle.

**Comment by csarven** [comment]
"That" was a referencing caching. "Good geo-distributed caching" is nice to have but shouldn't be seen as a requirement to run the site off a Solid server. It is plumbing and not particularly relevant for this issue. What kind of traffic is the current site experiencing such that not having proper caching as you describe would be detrimental? How are you measuring and deciding on substantial, especially considering that the website is quite basic (for the foreseeable future)?

**Comment by RubenVerborgh** [comment]
If we host it from the US, it will be slow in Australia (and other locations). Bad advertising for a project that aims to be for everyone. Not just plumbing.

There exist websites to test loading speed from around the world. The effects are likely larger than you think (compare my website to yours for instance).

**Comment by RubenVerborgh** [comment]
> Quantify "slow".

As I told you, pick a tool of your choice and measure.

> I find it is more concerning that we can't show a simple site without centralised third-party intervention.

Of course we can.

But this has dragged on too long already; it wasn’t that important to me. Let’s just make the site works well, is all I wanted to say.

**Comment by csarven** [comment]
We've both intended as additive, so all is good.

**Comment by jeswr** [comment]
Adding a note to say that when this happens the list of Pod Providers on https://solidproject.org/users/get-a-pod should *defnitely* be stored in RDF format so that other apps have access to a trusted and up-to-date list of Pod Providers that they can use to generate sign-in redirect pages from.
